.github/workflows/validate.yml

```diff
@@ -92,11 +92,15 @@ jobs:
       - uses: actions/checkout@v4
 
       # See https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#hackage-revisions
-      - name: Manually supplied constraints/allow-newer
-        if: github.event_name == 'workflow_dispatch'
+      - name: Add manually supplied constraints
```

Comment:
Are these two comments reversed?

Reply:
:facepalm: Thanks for looking after me @geekosaur :upside_down_face: